chat-widget/src/plugins/newMessageEventHandler.ts

```diff
@@ -20,15 +20,13 @@ export const createOnNewAdapterActivityHandler = (chatId: string, userId: string
     const raiseMessageEvent = (activity: IActivity) => {
         if (activity?.type === Constants.message) {
             const payload = {
+                // To identify hidden contents vs no content
                 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                text: (activity as any)?.text,
-                id: activity?.id,
+                text: (activity as any)?.text?.length >= 1 ? "*contents hidden*" : "",
```

Comment:
Can we add length of text as well?” Content hidden (20 chars)”

Reply:
![image](https://user-images.githubusercontent.com/45749980/178080716-cd9a1cf8-8712-4b5b-b785-93792ac72a91.png)